packages/ui/src/molecules/AggregateRating/stories/AggregateRating.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import type { Story } from '@storybook/react'
+import type { FC } from 'react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { AggregateRatingProps, RatingItemProps } from '../AggregateRating'
+import Component from '../AggregateRating'
+import mdx from './AggregateRating.mdx'
+
+const RatingIcon: FC<RatingItemProps> = (props) => {
+  const fillColor = {
+    empty: 'transparent',
+    partial: 'url(#partial)',
+    full: '#ffb100',
+  }
+
+  return (
+    <svg
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 32 32"
+      {...props}
+    >
+      <defs>
+        <linearGradient id="partial">
+          <stop offset="50%" stopColor="#ffb100" />
+          <stop offset="50%" stopColor="transparent" stopOpacity="1" />
+        </linearGradient>
+      </defs>
+      <path
+        d="M20.388,10.918L32,12.118l-8.735,7.749L25.914,
+             31.4l-9.893-6.088L6.127,31.4l2.695-11.533L0,
+             12.118l11.547-1.2L16.026,0.6L20.388,10.918z"
+        fill={fillColor[props['data-store-aggregate-rating-item']!]}
```

Comment:
What do you think about typing the props to avoid using the assertion operator (`!`)